ap.md

```diff
@@ -0,0 +1,256 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+<div class="large-12 columns">
+# Build Your Own Internet
+</div>
+<div class="large-7 small-12 columns">
```

Comment:
Hmm, in this case perhaps a single-column format will serve better; it's tough to read the diagrams when they're small, and they might help break up the text.